addons/project/models/project.py

```diff
@@ -1953,6 +1953,14 @@ def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=Fals
         self._ensure_fields_are_accessible(fields_list)
         return super(Task, self).read_group(domain, fields, groupby, offset=offset, limit=limit, orderby=orderby, lazy=lazy)
 
+    @api.model
+    def search_read(self, domain=None, fields=None, offset=0, limit=None, order=None):
```

Comment:
IMO it would make more sense to modify the `_search`, because otherwise we have two different behavior in the search read and in the search (which makes no sense if the change is the correct behavior)